wp-post-meta-revisions.php

```diff
@@ -63,19 +63,19 @@ public function _add_metadata_preview_filter() {
 	 *
 	 * @param Post object $new_autosave The new post being autosaved.
 	 */
-	public function _wp_autosave_post_revisioned_meta_fields( $new_autosave ) {
+	public function wp_autosave_post_revisioned_meta_fields( $new_autosave ) {
 		/*
 		 * The post data arrives as either $_POST['data']['wp_autosave'] or the $_POST
 		 * itself. This sets $posted_data to the correct variable.
 		 */
-		$posted_data = isset( $_POST['data']['wp_autosave'] ) ? $_POST['data']['wp_autosave'] : $_POST; // WPCS: CSRF ok. input var ok. sanitization ok.
+		$posted_data = isset( $_POST['data']['wp_autosave'] ) ? $_POST['data']['wp_autosave'] : $_POST; // phpcs:ignore WordPress.Security
```

Comment:
Suggestion: you may want to document the reason for ignoring this for the next person who looks at the code.

Two possible ways to do this:
`// phpcs:ignore Stnd.Cat.Sniff -- For reasons`

Or you could use disable/enable instead:
```php
		/*
		 * The post data arrives as either $_POST['data']['wp_autosave'] or the $_POST
		 * itself. This sets $posted_data to the correct variable.
		 *
		 * Ignoring ... for reasons...
		 * @phpcs:disable WordPress.Security
		 */
		$posted_data = isset( $_POST['data']['wp_autosave'] ) ? $_POST['data']['wp_autosave'] : $_POST;
		// phpcs:enable

Reply:
Good suggestion, will do.